osquery/sql/virtual_table.cpp

```diff
@@ -706,24 +708,21 @@ static int xBestIndex(sqlite3_vtab* tab, sqlite3_index_info* pIdxInfo) {
     for (size_t i = 0; i < static_cast<size_t>(pIdxInfo->nConstraint); ++i) {
       // Record the term index (this index exists across all expressions).
       const auto& constraint_info = pIdxInfo->aConstraint[i];
-#if defined(DEBUG)
-      plan("Evaluating constraints for table: " + pVtab->content->name +
-           " [index=" + std::to_string(i) +
-           " column=" + std::to_string(constraint_info.iColumn) +
-           " term=" + std::to_string((int)constraint_info.iTermOffset) +
-           " usable=" + std::to_string((int)constraint_info.usable) + "]");
-#endif
+      if (FLAGS_planner) {
+        plan("xBestIndex Evaluating constraints for table: " +
```

Comment:
`plan()` includes a check for `FLAGS_planner`.

Reply:
`xBestIndex` is very nuanced, thoughts on removing it from the log line?

Reply:
When I was debugging this behavior, I found it useful to know where (xBestIndex, xFilter, etc) a log line was coming from.  Let me know if you want it removed.